From 092d3cf24f0b2bed220e045d4e353b771882d544 Mon Sep 17 00:00:00 2001
From: Vanessa Maegima <vanessa.maegima@foundries.io>
Date: Thu, 15 Apr 2021 13:48:27 -0300
Subject: [PATCH 074/233] [FIO internal] power: pca9450: Add print after PMIC
 init

Add a print information in the pmic_init function for easier spotting
of which PMIC is being used in the imx8mm_evk boards.

Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>
---
 drivers/power/pmic/pmic_pca9450.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/power/pmic/pmic_pca9450.c b/drivers/power/pmic/pmic_pca9450.c
index 8c4d0a92306..5b15d8c5a38 100644
--- a/drivers/power/pmic/pmic_pca9450.c
+++ b/drivers/power/pmic/pmic_pca9450.c
@@ -27,5 +27,7 @@ int power_pca9450_init(unsigned char bus, unsigned char addr)
 	p->hw.i2c.tx_num = 1;
 	p->bus = bus;
 
+	printf("power_pca9450b_init\n");
+
 	return 0;
 }
-- 
2.40.1

